reasoner/resolution/resolver/ConjunctionResolver.java

```diff
@@ -228,18 +212,13 @@ private ResponseProducer mayUpdateAndGetResponseProducer(Request fromUpstream, i
     @Override
     protected ResponseProducer responseProducerCreate(Request fromUpstream, int iteration) {
         LOG.debug("{}: Creating a new ResponseProducer for request: {}", name(), fromUpstream);
-
-//        ResourceIterator<AnswerState.UpstreamVars.Derived> upstreamAnswers = toUpstreamAnswers(
-//                fromUpstream, compatibleBoundAnswers(conceptMgr, conjunction, fromUpstream.partialAnswer().conceptMap()));
-//
         Plans.Plan plan = plans.getOrCreate(fromUpstream.partialAnswer().conceptMap().concepts().keySet(), resolvables, negateds);
```

Comment:
while we're here:
arguments for doing traversals in conjunctions if there are no concludables or negations:
1. saves doing messages to the child `Retrievable`

2. saves saving all the deduplicated answers in both the `Retrievable` and the `Root.X` (root must have it anyway, I suppose we could disable it if there's no negation or concludable?). NOTE: don't need deduplication set in retrievable at all - traversal engine already guarantees distinctness!